lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -433,7 +433,7 @@
             <?php if ($register_voters) : ?>
               <p><?php print $voter_reg_share_modal_text_2 ?></p>
             <?php endif; ?>
-            <?php if ($show_social_share_link) : ?>
+            <?php if ($show_social_share_link == 1) : ?>
```

Comment:
hmm I guess it is a bit telling by the name of the variable. I think just more obvious with this unless @sbsmith86 I'm missing something else?